.github/workflows/dev-pr.yml

```diff
@@ -0,0 +1,39 @@
+name: pr-submit
+on:
+  pull_request:
+    branches: [ dev-0.8.0]
```

Comment:
this will 'never' run in 'develop' branch. (and this commit is for develop branch).